python/ray/serve/api.py

```diff
@@ -801,6 +801,58 @@ def get_deployment_handle(
         RayServeException: If no Serve controller is running, or if
             calling from outside a Serve application and no application
             name is specified.
+
+    The following example gets the handle to the ingress deployment of
+    an application, which is equivalent to using `serve.get_app_handle`.
+
+    .. code-block:: python
```

Comment:
I think in order for those code to be tested in doc test, we should add `>>>` in front of the code line instead. 
Re: [this doc](https://docs.ray.io/en/latest/ray-contribute/writing-code-snippets.html?highlight=code-block#doctest-style-examples) Tip `If you’re writing docstrings, exclude doctest:: to simplify your code.`
```
Example:
    >>> def is_even(x):
    ...     return (x % 2) == 0
    >>> is_even(0)
    True
    >>> is_even(1)
    False
```

Reply:
Thanks @GeneDer! I switched to `testcode`, and confirmed that the code is being tested